controllers/roomController.js

```diff
@@ -4,4 +4,20 @@ const {
 
 exports.createRoomId = function(req, res) {
     res.redirect(`/users/room/${uuidv4()}`);
+};
+
+exports.uploadCallRecording = function(req, res) {
+    const params = {
+        Bucket: BUCKET_NAME,
+        Key: 'cat.jpg', // File name you want to save as in S3
+        Body: fileContent
+    };
+
+    // Uploading files to the bucket
+    s3.upload(params, function(err, data) {
+        if (err) {
+            throw err;
```

Comment:
instead of throwing error, wouldn't it be better to send a json response with
`
{
  success: false,
  errorMessage: err.message,
`
?